### ISSUE TITLE
How to set httpInterceptor config parameter to false?

### ISSUE THREAD

**Issue by p-p-m** [sahat/satellizer#250]
I do not want to set auth header. But I can not disable adding satellizer function to $httpProvider.interceptors:

I tried to do this:
```
angular.module('myapp')
  .config(['satellizer.config', function(config) {
    config.httpInterceptor = false;
  }]);
```
And this:
```
angular.module('myapp')
  .config(['satellizer.config', function(config) {
    config.httpInterceptor = false;
  }]);
```
But satellizer adds his funtion to $httpProvider.interceptors before my code execution.

**Comment by sahat** [comment]
You shouldn't be injecting any services prefixed with Satellizer word. Those are internal services.

Use $authProvider.httpIntercetor= false

**Comment by p-p-m** [comment]
Thanks for your answer, but I tried to do as you recomend and it didn't help.
I added such config to my project:

```
(function() {
  angular.module('myapp')
    .config(['ENV', '$authProvider', initAuthProvider]);

    function initAuthProvider(ENV, $authProvider) {

      $authProvider.facebook({
        clientId: ENV.facebookClientId,
        url: ENV.apiEndpoint + ENV.facebookEndpointUrl
      });

      $authProvider.google({
        clientId: ENV.googleClientId,
        url: ENV.apiEndpoint + ENV.googleEndpointUrl
      });

      $authProvider.httpIntercetor = false;

    }

})();
```
But this code is executed before my .config 
https://github.com/sahat/satellizer/blob/master/satellizer.js#L590-L609
and it adds auth header.

Where should I put `$authProvider.httpIntercetor= false` ?

**Comment by sahat** [comment]
That line has to be executed during the `config` phase, not before or after. See [example](https://github.com/sahat/satellizer/blob/master/examples/client/app.js).

**Comment by sahat** [comment]
Fixed in #256.
